openstack/css/v1/clusters/UpdateClusterFlavor.go

```diff
@@ -0,0 +1,108 @@
+package clusters
+
+import (
+	"fmt"
+
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/build"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/css/v1/flavors"
+)
+
+const (
+	DefaultNodeType = "ess"
+)
+
+type ClusterFlavorOptsBuilder interface{}
+
+// ClusterFlavorOpts for updating the flavor of a cluster.
+type ClusterFlavorOpts struct {
+	// Whether to verify replicas.
+	NeedCheckReplica bool `json:"needCheckReplica"`
+	// ID of the new flavor.
+	Flavor string `json:"-" required:"true"`
+}
+
+// ClusterNodeFlavorOpts for updating the flavor of a specific cluster node type.
+type ClusterNodeFlavorOpts struct {
+	// Whether to verify replicas.
+	NeedCheckReplica bool `json:"needCheckReplica"`
+	// ID of the new flavor.
+	Flavor string `json:"-" required:"true"`
+	// Type of the cluster node to modify.
+	Type string `json:"type" required:"true"`
+}
+
+type apiFlavorOpts struct {
+	// Whether to verify replicas.
+	NeedCheckReplica bool `json:"needCheckReplica"`
+	// ID of the new flavor.
+	NewFlavorID string `json:"newFlavorId"`
+}
+
+// getFlavorIDByName retrieves the flavor ID for a given flavor name and node type.
+func getFlavorIDByName(client *golangsdk.ServiceClient, flavorName, nodeType string) (string, error) {
```

Comment:
no need to use otcextensions arch here and behaviour of funcs, it would bring issues in terraform

Reply:
Function is removed.